nginx.conf

```diff
@@ -0,0 +1,78 @@
+user  nginx;
```

Comment:
I guess you just copied the nginx.conf from the PR check env, right?
I'd ask you to distill it down to the bare minimum of what is needed (maybe just the proxy rules?)